example/rcnn/rcnn/warmup.py

```diff
@@ -0,0 +1,59 @@
+from mxnet.lr_scheduler import LRScheduler
```

Comment:
@sxjscience yes, i think so.   i write warmup because there is a backward grad bug(not be zero) in costum op and this will bring nan during training. now this has been fixed, so warm up is no need, and i found it is even bad for training(i forget to revise that at last pr, but i do not use it when training face detection).